components/Badge/Badge.js

```diff
@@ -23,13 +23,13 @@ function Badge({ className, icon, isImageFirst, label }) {
     <div className={classNames(styles.Badge, className)}>
       {isImageFirst ? (
         <>
-          <div className={styles.iconContainer}>{icon}</div>
-          <span className={styles.label}>{label}</span>
+          <figure className={styles.iconContainer}>{icon}</figure>
```

Comment:
so `<figcaption>` is meant to be a child of `<figure`, not a sibling.

Basically, I'd expect a figure instead of a React.Fragment, and then the div can stay behind. You did the figcaption in the right spot tho!

Reply:
Refactored Badge